app/Helpers/MiscHelper.php

```diff
@@ -2,20 +2,20 @@
 
 namespace App\Helpers;
 
-class MiscHelper{
+class MiscHelper {
```

Comment:
__[code_sniffer]__
* Opening brace of a class must be on the line after the definition

<!-- Comment by [SideCI](https://sideci.com) -->